Desktop/data/expanddataproxymodel.h

```diff
@@ -32,19 +148,33 @@ class ExpandDataProxyModel : public QObject
 	void				removeRow(int row);
 	void				removeColumn(int col);
 	void				insertRow(int row);
-	void				insertColumn(int col);
-	QString				insertColumnSpecial(int col, bool computed, bool R);
+	void				insertColumn(int col, bool computed, bool R);
```

Comment:
I called it "special" cause it doesnt take into account the proxymodels possible columnfiltering.
This wont bite us now but it is good to keep it in mind.